python/hail/docs/tutorials/hail-overview.ipynb

```diff
@@ -970,11 +970,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## We've been confounded!\n",
+    "## Confounded!\n",
     "\n",
-    "This q-q plot is very inflated. This is a problem, and indicates the presence of a confounder. \n",
+    "The observed p-values drift away from the expectation immediately. Either every SNP in our dataset is causally linked to caffeine consumption (unlikely), or there's a confounder.\n",
     "\n",
-    "We didn't tell you, but reported sample ancestry was actually used to simulate this phenotype. We need to include it as a covariate. The best way to do that is by including computed principal components in our model.\n",
+    "We didn't tell you, but sample ancestry was actually used to simulate this phenotype. This leads to a [stratified](https://en.wikipedia.org/wiki/Population_stratification) distribution of the phenotype. We need to include ancestry as a covariate. We have reported ancestry as sample annotations, but we may not trust these metrics The best way to do that is by including computed principal components in our model.\n",
```

Comment:
Period missing after "metrics". Also, by inserting the "We have reported sample ancestry... metrics" sentence, you've made the following sentence's use of the word "that" a little awkward, since "that" refers to the including of ancestry as a covariate.